File: backend/infrastructure/models/History.cs
namespace infrastructure.models;

public class History
{
    public int HistoryId { get; set; }
    public string? UserEmail { get; set; }
    public int UnitId { get; set; }
    public DateTime Date { get; set; }
    public EventType EventTypeId { get; set; }
}

public enum EventType
{
    Open,
    Close,
    AlarmTriggered,
    AlarmStopped,
    AlarmArmed,
    AlarmDisarmed,
}

File: frontend/src/models/History.ts
export interface History{
  historyId?: number;
  userId: number;
  unitId: number;
  date: Date;
  EventType: EventType;
}

export enum EventType{
  Open,
  Close,
  AlarmTriggered,
  AlarmStopped,
  AlarmArmed,
  AlarmDisarmed,
}
